Sources/NIOPosix/Bootstrap.swift

```diff
@@ -712,9 +712,14 @@ private extension Channel {
 
 /// A `ClientBootstrap` is an easy way to bootstrap a `SocketChannel` when creating network clients.
 ///
-/// Usually you re-use a `ClientBootstrap` once you set it up and called `connect` multiple times on it.
+/// You may re-use a `ClientBootstrap` once you set it up and called `connect` multiple times on it.
```

Comment:
while we’re at it, can we fix the grammar *called* → *call*?

Reply:
```suggestion
/// You may re-use a `ClientBootstrap` once you set it up and call `connect` multiple times on it.
```